### ISSUE TITLE
How to Create a Study Routine that Works?

### ISSUE THREAD

**Issue by Williamsds** [Williamsds/paysomeonetodoyourassignment#1]
Do you procrastinate when it’s study time? Are you finding it difficult to keep up with the subjects you need to study? Do you stay awake all night cramming as much as you can the night before an exam? If the answer to these questions is yes, then you need to create a study plan or routine that works.
A good study routine will prevent you from <a href="https://www.mindtools.com/pages/article/newHTE_96.htm">procrastinating</a> and help you achieve your academic goals faster and easier. How do you create an effective study routine? This article is going to answer this question.
1.	Discover your learning style
Do you know how you learn? If you don’t know, how will you improve your learning methods? There different types of learning styles. Visual learning involves the use of pictures to help you understand and remember things quickly. 
Aural learning involves learning through hearing and listening attentively. Verbal learning involves using words and speech to accelerate learning. Physical learning involves using your body and focusing on your feelings to learn. 
Logical learning involves the use of systems and structures to learn. Which learning method works best for you? Understanding your learning style will put you ahead of the crowd.
2.	Set realistic goals
A goal can only be achieved through planning and execution. If you don’t execute your plan, then the plan is meaningless. You can only achieve your goals if the planning and execution is done perfectly. 
While being ambitious will help you perform well, it’s important to set realistic goals. For instance, reading and reviewing 30 pages of a book in one night is ambitious but not realistic. When you set such goals, you are only setting yourself up for failure.  
Always remember, less is more. Set simple goals and most importantly, execute. If you don’t’ have time, you can <a href="https://australianwritings.com.au/pay-someone-to-do-your-assignment/">pay someone to do your assignment</a>. The more you achieve small goals, the easier it will be for you to set big goals.
3.	Create a to do list every day
A to do list will help you save a lot of time. Write down all the things you have to do in school today. After writing them down, arrange the activities in order of priority. 
Start working on the first task and don’t go to the next one until you’ve completed it. By doing this, you’ll never miss deadlines and you’ll get good grades.
4.	Create a timetable or schedule
You have to create a timetable before starting your study sessions. Students who don’t have a study timetable waste a lot of time because they have no idea what and when they’ll be studying. 
Effective students map out everything in advance. And they always know what they’ll be doing during their study sessions. They set <a href="https://www.actionforhappiness.org/take-action/set-your-goals-and-make-them-happen">goals</a> for every study session. 
On the other hand, unsuccessful students tend to postpone their study sessions because they have no goals. Studying regularly and effectively will save you a lot of time and energy in the long run.
